SETUP/upgrade/21/20240414_populate_latest_shapshot_tallies.php

```diff
@@ -0,0 +1,173 @@
+<?php
```

Comment:
Typo in name of script? 
SETUP/upgrade/21/20240414_populate_latest_shapshot[** snapshot]_tallies.php

Reply:
Good catch! I've renamed this script after merging `last_snapshot_tallies` into `current_tallies`.